tomviz/Utilities.cxx

```diff
@@ -762,6 +764,77 @@ QWidget* mainWidget()
   return pqCoreUtilities::mainWidget();
 }
 
+void rasterTransferFunction2DBox(vtkImageData* histogram2D, const vtkRectd& box,
```

Comment:
Why add this to Utilities? It seems very histogram specific. Also, I don't see where this was moved from/deletion of equivalent code. Maybe I am missing something here?

Reply:
Oops.  It moved to Utilities because it needed it in ModuleVolume's load state.

Reply:
I forgot to delete the original.